package.json

```diff
@@ -45,6 +45,8 @@
 	"overrides": {
 		"ngrok": {
 			"jsprim": "1.4.2"
-		}
```

Comment:
Is this still needed after `ngrok` update?

Reply:
No, you're right, that dependency should no longer be there. Thanks!